vendor/assets/javascripts/highcharts/stock/highcharts-more.js

```diff
@@ -0,0 +1,56 @@
+this.chart.axisOffset[this.side]=0;this.center=this.pane.center=T.getCenter.call(this.pane)},getLinePath:function(a,b){var c=this.center,b=o(b,c[2]/2-this.offset);return this.chart.renderer.symbols.arc(this.left+c[0],this.top+c[1],b,b,{start:this.startAngleRad,end:this.endAngleRad,open:!0,innerR:0})},setAxisTranslation:function(){G.setAxisTranslation.call(this);if(this.center)this.transA=this.isCircular?(this.endAngleRad-this.startAngleRad)/(this.max-this.min||1):this.center[2]/2/(this.max-this.min||
+1),this.minPixelPadding=this.isXAxis?this.transA*this.minPointOffset:0},beforeSetTickPositions:function(){this.autoConnect&&(this.max+=this.categories&&1||this.pointRange||this.closestPointRange||0)},setAxisSize:function(){G.setAxisSize.call(this);if(this.isRadial){this.center=this.pane.center=n.CenteredSeriesMixin.getCenter.call(this.pane);if(this.isCircular)this.sector=this.endAngleRad-this.startAngleRad;this.len=this.width=this.height=this.center[2]*o(this.sector,1)/2}},getPosition:function(a,
+b){return this.postTranslate(this.isCircular?this.translate(a):0,o(this.isCircular?b:this.translate(a),this.center[2]/2)-this.offset)},postTranslate:function(a,b){var c=this.chart,f=this.center,a=this.startAngleRad+a;return{x:c.plotLeft+f[0]+Math.cos(a)*b,y:c.plotTop+f[1]+Math.sin(a)*b}},getPlotBandPath:function(a,b,c){var f=this.center,e=this.startAngleRad,d=f[2]/2,h=[o(c.outerRadius,"100%"),c.innerRadius,o(c.thickness,10)],j=/%$/,m,g=this.isCircular;this.options.gridLineInterpolation==="polygon"?
+f=this.getPlotLinePath(a).concat(this.getPlotLinePath(b,!0)):(a=Math.max(a,this.min),b=Math.min(b,this.max),g||(h[0]=this.translate(a),h[1]=this.translate(b)),h=R(h,function(a){j.test(a)&&(a=A(a,10)*d/100);return a}),c.shape==="circle"||!g?(a=-Math.PI/2,b=Math.PI*1.5,m=!0):(a=e+this.translate(a),b=e+this.translate(b)),f=this.chart.renderer.symbols.arc(this.left+f[0],this.top+f[1],h[0],h[0],{start:Math.min(a,b),end:Math.max(a,b),innerR:o(h[1],h[0]-h[2]),open:m}));return f},getPlotLinePath:function(a,
+b){var c=this,f=c.center,e=c.chart,d=c.getPosition(a),h,j,m;c.isCircular?m=["M",f[0]+e.plotLeft,f[1]+e.plotTop,"L",d.x,d.y]:c.options.gridLineInterpolation==="circle"?(a=c.translate(a))&&(m=c.getLinePath(0,a)):(s(e.xAxis,function(a){a.pane===c.pane&&(h=a)}),m=[],a=c.translate(a),f=h.tickPositions,h.autoConnect&&(f=f.concat([f[0]])),b&&(f=[].concat(f).reverse()),s(f,function(d,b){j=h.getPosition(d,a);m.push(b?"L":"M",j.x,j.y)}));return m},getTitlePosition:function(){var a=this.center,b=this.chart,
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.